generators/getMetricFunctions.js

```diff
@@ -0,0 +1,52 @@
+const metricsList = [
```

Comment:
file names should not be camelCase
https://stackoverflow.com/questions/7273316/what-is-the-javascript-filename-naming-convention

Reply:
Almost all of our files in SanSheets are camelCase :grinning: